Sources/Parsers/StoryboardParser.swift

```diff
@@ -28,90 +28,70 @@ public final class StoryboardParser {
     let customModule: String?
   }
 
+  enum XMLScene {
```

Comment:
Yeah, not that big a deal imho, as long as we use `…Path`, `…Tag` & `…Attribute` suffixes, we don't have that many constants here to justify having structured constants at all cost, that's ok